MdsSupportingClasses/MdsColliveryService.php

```diff
@@ -1030,11 +1040,17 @@ public function returnDefaultAddress()
             $data = [
                 'address' => $default_address,
                 'default_address_id' => $default_address_id,
-                'contacts' => $this->collivery->getContacts($default_address_id),
             ];
 
+            if (!isset($default_address['contacts'])) {
+                $data['contacts'] = $this->collivery->getContacts($default_address_id);
+            } else {
+                $data['contacts'] = $default_address['contacts'];
+            }
+
             return $data;
         } catch (SoapConnectionException $e) {
+            new SoapConnectionException($e->getMessage());
```

Comment:
New API doesnt use SOAP

Reply:
Can I just rename the class to something else and keep the logs? 
Or is there a different exception log you'd want me to use instead?

Reply:
See comments above regarding not catching that exception and using login directly.

Reply:
Updated these to use a new CurlConnectionException instead, that functions in a similar way to how the SoapConnectionException function use to work.